src/plugins/simple-pricing/resolvers/ProductVariant.js

```diff
@@ -0,0 +1,7 @@
+import xformProductVariantPricing from "../xforms/xformProductVariantPricing.js";
+import xformProductVariantPrice from "../xforms/xformProductVariantPrice.js";
+
+export default {
+  price: (node) => xformProductVariantPrice(node),
```

Comment:
Can we add a resolver for `compareAtPrice` that...

- Returns the `compareAtPrice` as is with no modification
- but, Logger.warn with a deprecation notice?

Reply:
updated